R/shiny_prerendered.R

```diff
@@ -431,6 +431,9 @@ shiny_prerendered_extract_context <- function(html_lines, context) {
     if (in_context)
       context_lines <- c(context_lines, html_lines[[i]])
   }
+
+  # unescape code, see https://github.com/rstudio/rmarkdown/issues/943
+  context_lines <- gsub("<\\u002f", "</", context_lines, fixed = TRUE)
```

Comment:
Just a note about this: I believe that unescaping is unnecessary, but it also shouldn't cause any harm.